README.md

```diff
@@ -1,6 +1,14 @@
 # UX Team Intern Onboarding Project
 UX Team Intern onboarding project.
-Testing SSH Connection
 # How To Use
-
+Run npm install to install dependencies
+```
+$ npm install 
```

Comment:
Should be 
```suggestion
$ yarn install 
```

Using both `npm` and `yarn` can cause problems, as they use slightly different lock file. You'll want to make sure your consistently using `yarn`

Reply:
Got it